rpcs3/Emu/Cell/Modules/cellSaveData.cpp

```diff
@@ -1360,6 +1360,20 @@ static NEVER_INLINE error_code savedata_op(ppu_thread& ppu, u32 operation, u32 v
 					continue; // system files are not included in the file list
 				}
 
+				if (!g_cfg.misc.allow_filesystem_metadata_in_save_folders && entry.name.starts_with("._"sv))
```

Comment:
Why do we still need the cfg option if you check for header magic anyway?

Reply:
Because it is still valid as ps3 savedata, and I don't care even if 0 games use such file format.

Reply:
It needs moving to check_filename.